### ISSUE TITLE
This Channel Is Not Responding Error

### ISSUE THREAD

**Issue by ExtraKiller** [trakt/Plex-Trakt-Scrobbler#362]
Hi, I seem to have an issue with "This Channel Is Not Responding" 
I deleted the trakt.db and nothing changed.
Here is my log file: [com.plexapp.plugins.trakttv.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/502560/com.plexapp.plugins.trakttv.txt)

Any help?

**Comment by outcorpse** [comment]
Same problem here

**Comment by fuzeman** [comment]
@ExtraKiller Looks like a network error occurred (out of my control):
```
Max retries exceeded with url: /sync/ratings/shows (Caused by ProtocolError('Connection aborted.', error("(10054, 'WSAECONNRESET')",))
```

Is this issue still happening? does it happen every time you trigger a sync?

---------------

@outcorpse Post the latest plugin [log file](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files) if you want me to have a look at your issue, many problems can cause the "This Channel Is Not Responding" error.

**Comment by ExtraKiller** [comment]
Yes, it happens everytime I trigger the sync, in a few hours I'll upload the recent log. Don't know what may cause it...

**Comment by ExtraKiller** [comment]
@fuzeman 
[com.plexapp.plugins.trakttv.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/519237/com.plexapp.plugins.trakttv.txt)
Added...  Can u help?

**Comment by outcorpse** [comment]
When i added the Plex port to my routers firewall it looks like the problem
is being resolved. Will let you know. Thanks for the help. :-)

**Comment by phuture2k** [comment]
Same problem here. 
[com.plexapp.plugins.trakttv.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/528391/com.plexapp.plugins.trakttv.txt)

**Comment by fuzeman** [comment]
@ExtraKiller @phuture2k 

It looks like you are running a version of the plugin which is probably over 5 months old *(guessing based on the log file generated)*. Try updating to the latest release ([v1.1.0.6](https://github.com/trakt/Plex-Trakt-Scrobbler/releases/tag/v1.1.0.6)) to see if this issue has already been fixed.

**Comment by GloveNinja217** [comment]
Hi there,

I just installed this plug-in and I'm having the same problem. I'm running Windows 10 on my computer that's the Plex Media Server. I just installed the plug-in from this page: https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Installation%20-%20Windows following the instructions.

I went into Plex and authenticated into my Trakt account and the only setting I changed was to disable syncing Ratings. I went to run a full sync and I received the "This channel is not responding" error. 

If you can direct me on how to get the log files I will submit them to this thread as well.

**Comment by ExtraKiller** [comment]
@fuzeman I did a clean install of the PC and installed the one you gave a link to, still the same

[com.plexapp.plugins.trakttv.txt](https://github.com/trakt/Plex-Trakt-Scrobbler/files/535421/com.plexapp.plugins.trakttv.txt)

@GloveNinja217 drag the file to the comment box

**Comment by ExtraKiller** [comment]
@fuzeman, anything new? sorry for the bump.

**Comment by fuzeman** [comment]
I've made a minor change to the sync queuing system to try avoid these errors from being displayed, install the development branch *([download](https://github.com/trakt/Plex-Trakt-Scrobbler/archive/develop.zip))* and let me know if this fixes the issue.

**Comment by smithinoz** [comment]
Hi similar issue here, clean install logged in to plex account , trakt account, new pin requests Generate account not authorised errors.
Last worked about 3 months ago, before the windows 10 update 1
Tried disabling windows f/w

**Comment by fuzeman** [comment]
@smithinoz Could you open a new issue and upload the latest plugin log files for me to review, see [this page](https://github.com/trakt/Plex-Trakt-Scrobbler/wiki/Log-files) for more details.

**Comment by fuzeman** [comment]
I've released some fixes for this issue in [v1.2.0](https://github.com/trakt/Plex-Trakt-Scrobbler/releases/tag/v1.2.0), please post the latest plugin log file if you experience this issue again.
